packages/flutter/lib/src/material/icons.dart

```diff
@@ -21,6 +21,37 @@ import 'package:flutter/widgets.dart';
 ///   uses-material-design: true
 /// ```
 ///
+/// {@tool sample}
+/// This example shows how to use [Icons] to create [Row] of icons in different
+/// colors and sizes.
```

Comment:
Could also add something along the lines of "The first of which uses a [semanticLabel] to announce in accessibility modes like TalkBack and VoiceOver."